plugins/utils/better-escape.nix

```diff
@@ -0,0 +1,65 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.better-escape;
+  helpers = import ../helpers.nix {inherit lib;};
+in {
+  options.plugins.better-escape =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "better-escape";
+
+      package = helpers.mkPackageOption "better-escape" pkgs.vimPlugins.better-escape-nvim;
+
+      mapping = helpers.mkNullOrOption (with types; listOf str) ''
+        List of mappings to use to enter escape mode.
+      '';
+
+      timeout = helpers.defaultNullOpts.mkInt 100 ''
```

Comment:
by default this uses a value from `vim.o.timeoutlen`, I set this to a 100 just to create the PR. but is there a way to fetch this ? I saw something getting `vim.g.` values

Reply:
I think I fixed this